lib/svg-helpers.js

```diff
@@ -1,5 +1,7 @@
 function svg2base64(svg) {
-  return `data:image/svg+xml;base64,${Buffer.from(svg).toString('base64')}`
+  return `data:image/svg+xml;base64,${Buffer.from(svg.trim()).toString(
```

Comment:
The difference in the logo tests was a `\n` vs `\r\n` issue. I decided that really the best thing to do was just strip any kind of trailing whitspace rather than try to normalise it as any trailing whitspace after the `>` is doing nothing of use anyway.